argusclient/client.py

```diff
@@ -416,7 +416,18 @@ def add(self, alert):
             alertobj.trigger = alertobj.triggers.add(alert.trigger)
         if alert.notification:
             alertobj.notification = alertobj.notifications.add(alert.notification)
-        if alert.trigger and alert.notification:
+        elif alert.notifications and isinstance(alert.notifications, list):
```

Comment:
The `notifications` member of a new `alert` should never be anything other than a `list` type. It is worth establishing this at the beginning of the method and drop the check here. Also, what if both `notification` and `notifications` members exist?